keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -374,7 +381,9 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
-A new node_e2e test with  `serialize-image-pulls==false` will be added to make sure that when maxParallelImagePulls is reached, all further image pulls will be blocked.
+A new node_e2e test with  `serialize-image-pulls==false` will be added test parallel image pull limits. 
```

Comment:
L368 - please just say that we don't need integration tests - e2e tests will cover all we need